tqdm.py

```diff
@@ -0,0 +1,42 @@
+import cardinality  # You might need to do 'pip install cardinality'.
+import time
+
+PROGRESS_BAR_LEN = 20
+
+
+class Tqdm(object):
+
+    def __init__(self, iterable):
+        self.iterable = iterable
+        self.length = cardinality.count(iterable)
+
+    def __iter__(self):
+        return TqdmIterator(iter(self.iterable), self.length)
+
+
+class TqdmIterator(object):
+
+    def __init__(self, iterator, length):
+        self.start_time = time.time()
```

Comment:
Time should be measured only from the first call to `__next__` (for example, if I define the iterator at the beginning of my program, and use it only after a while).